### PR TITLE
Webpack changes

### PR DESCRIPTION
**Pull request by nkomonen-amazon**
## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by justinmk3** [comment]
package test failure looks legitimate:

```
  Module not found: Error: Can't resolve './src/extensionWeb.ts' in '/codebuild/output/src2388414412/src/github.com/aws/aws-toolkit-vscode/packages/amazonq'
  resolve './src/extensionWeb.ts' in '/codebuild/output/src2388414412/src/github.com/aws/aws-toolkit-vscode/packages/amazonq'
    using description file: /codebuild/output/src2388414412/src/github.com/aws/aws-toolkit-vscode/packages/amazonq/package.json (relative path: .)
      Field 'browser' doesn't contain a valid alias configuration
```

**Review by justinmk3 — APPROVED** [review]

File: packages/toolkit/package.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
         "lint": "echo 'Nothing to lint yet!'",
         "createRelease": "ts-node ../../scripts/createRelease.ts",
         "newChange": "ts-node ../../scripts/newChange.ts",
-        "watch": "npm run clean && npm run buildScripts -- --vueHr && tsc -watch -p ./",
+        "watch": "npm run clean && npm run buildScripts && tsc -watch -p ./",
         "serve": "webpack serve --config-name mainServe --mode development",
```
Comment (hayemaxi) [inline packages/toolkit/package.json:73]: mainServe no longer exists with the other changes

File: packages/toolkit/package.json [hunk 2]
```diff
@@ -57,10 +57,10 @@
         "buildScripts": "npm run generateNonCodeFiles && npm run copyFiles",
         "generateNonCodeFiles": "ts-node ../../scripts/generateNonCodeFiles.ts",
         "copyFiles": "ts-node ./scripts/build/copyFiles.ts",
-        "clean": "ts-node ../../scripts/clean.ts dist/ LICENSE NOTICE quickStart*",
-        "compile": "npm run clean && npm run buildScripts && webpack --mode development",
-        "webWatch": "npm run clean && npm run copyFiles --vueHr && webpack --config-name web --watch",
-        "webCompile": "npm run clean && npm run buildScripts -- --vueHr && webpack --config-name web",
+        "clean": "ts-node ../../scripts/clean.ts dist/ LICENSE NOTICE quickStart* CHANGELOG.md",
```
Comment (hayemaxi) [inline packages/toolkit/package.json:60]: we don't want to delete CHANGELOG.md

File: packages/toolkit/.vscode/tasks.json
Comment (hayemaxi) [inline packages/toolkit/.vscode/tasks.json:1]: i think we will need to copy these tasks tot he amazonq debug as well

**Pull request merged**